src/compiler/compiler.rs

```diff
@@ -390,7 +402,7 @@ where
 {
     let mut path_transformer = dist::PathTransformer::new();
     let compile_commands = compilation
-        .generate_compile_commands(&mut path_transformer)
+        .generate_compile_commands(&mut path_transformer, true)
```

Comment:
In the local case we want this to be `false`, no? Does it matter?

Reply:
In that case, is a separate preprocessing step done as well? If yes, we should definitely enable the behaviour. This won't run into the glibc issue then IIUC. Otherwise, it doesn't matter.

Reply:
It is. Good point.